docs/source/app-dev/services.rst

```diff
@@ -56,9 +56,12 @@ A call to the command submission service will return as soon as the ledger serve
 
 The on-ledger effect of the command execution will be reported via the `transaction service <#transaction-service>`__, described below. The completion status of the command is reported via the `command completion service <#command-completion-service>`__. Your application should receive completions, correlate them with command submission, and handle errors and failed commands. Alternatively, you can use the `command service <#command-service>`__, which conveniently wraps the command submission and completion services.
 
-Commands can be labeled with two application-specific IDs, both of which are returned in completion events:
+Each intended ledger change is identified by the **change ID**: the triplet of the submitting :ref:`party <com.daml.ledger.api.v1.Commands.party>`, :ref:`application ID <com.daml.ledger.api.v1.Commands.application_id>` and :ref:`command ID <com.daml.ledger.api.v1.Commands.command_id>`.
```

Comment:
We take `actAs union party` as the submitting parties. Do you want to change this here, too?

Reply:
Is saying "the submitting parties" too vague?

Reply:
I find that the links to the corresponding protobuf fields help the reader. It is weird if two fields are linked and we just leave "submitting parties" unanchored.

Reply:
8b7eec5, let me know if it's clear or if you think that wording it directly as `actAs union party` with links is better.

Reply:
That's great. Thanks.